tests/compute/test_traversal.py

```diff
@@ -16,7 +16,7 @@ def toset(x):
     # F.zerocopy_to_numpy may return a int
     return set(F.zerocopy_to_numpy(x).tolist())
 
-def test_bfs(n=100):
+def test_bfs(n=100, gtype='int'):
```

Comment:
Use the `parametrize_dtype` decorator from `utils`. See `test_heterograph` for more details.